Reformat and update docs (#66)
primer/presentations: content/slide-layouts/quotes.mdx

---
title: Quotes
---

> Quotes are set up in two different layouts: left-aligned and centered. It is recommended that longer quotes be left-aligned and shorter quotes centered.

Reviewer: ```suggestion
Quotes are set up in two different layouts: left-aligned and centered. Longer quotes should be left-aligned and shorter quotes should be centered.
```
